bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -70,14 +70,18 @@ impl Opcode for Call {
             ),
             (
                 CallContextField::IsStatic,
-                (current_call.is_static as u64).into(),
+                if is_call {
```

Comment:
I think old one  `(current_call.is_static as u64).into(),` also work , do you have tested ?

Reply:
Yes. I tested that some case was failed if not set `is_static = 1` explicitly for `STATICCALL`. So added `is_call` flag and set explicitly.

Reply:
should fix `current_call.is_static` property to make it really stands for `is_static` meaning I think.

Reply:
Fixed to get `callee` flag `is_static`, and verify with `call_context_write`.